### DISCUSSION TITLE
How do I submit improvements?

### DISCUSSION THREAD

**Discussion by gitfool** [cake-build/cake#3313] [Q&A]
With the introduction of an issue template I can only submit a bug report. How do I submit improvements?

**Comment by augustoproiete** [comment]
Good point. We should update the issue template to include that. In general we've been recommending writing a post in the [Ideas](https://github.com/cake-build/cake/discussions/categories/ideas) section in GitHub Discussions to get buy-in, and then create an issue if make sense.

**Reply by gitfool** [reply]
I was going to submit a few version bumps... NuGet and .NET SDK etc. Shouldn't need a discussion?

**Reply by augustoproiete** [reply]
Yep, prob. discussion not needed. Feel free to create bug issue for now and one of us can update the tags
